lib/Jsrt/Jsrt.cpp

```diff
@@ -2475,6 +2476,22 @@ JsErrorCode RunScriptCore(const wchar_t *script, JsSourceContext sourceContext,
     });
 }
 
+JsErrorCode RunScriptCore(const char *script, JsSourceContext sourceContext, const char *sourceUrl, bool parseOnly, JsParseScriptAttributes parseAttributes, bool isSourceModule, JsValueRef *result)
+{
+    utf8::NarrowToWide url((LPCSTR)sourceUrl);
+    if (!url)
+    {
+        return JsErrorOutOfMemory;
+    }
+
+    return RunScriptCore(reinterpret_cast<const byte*>(script), strlen(script), LoadScriptFlag_Utf8Source, sourceContext, url, parseOnly, parseAttributes, isSourceModule, result);
+}
+
+JsErrorCode RunScriptCore(const wchar_t *script, JsSourceContext sourceContext, const wchar_t *sourceUrl, bool parseOnly, JsParseScriptAttributes parseAttributes, bool isSourceModule, JsValueRef *result)
```

Comment:
This is intentional, corresponding to public wchar_t version of JSRT APIs. We may need a cleanup to move all wchar_t versions under `#ifdef _WIN32`.